testdata/go1.24/struct_tag.go

```diff
@@ -21,7 +21,7 @@ type decodeAndValidateRequest struct {
 	OptionalQuery    string          `json:"-" querystring:"queryfoo"`
 	optionalQuery    string          `json:"-" querystring:"queryfoo"` // MATCH /tag on not-exported field optionalQuery/
 	// No-reg test for bug https://github.com/mgechev/revive/issues/208
-	Tiret       string `json:"-,"`
+	Tiret       string `json:"-,"`                   // MATCH /useless empty option for ignored field (remove the comma after -) in json tag/
```

Comment:
This one is valid, it shouldn't be reported

But this won't

```
`json:"-,omitempty"`
`json:"-,-"
```

There are legitimate uses case for using - as a field

https://github.com/search?q=language%3Ajson+%2F%22-%22%3A%2F&type=code

Among them, things around characters matching